Week4/BlockbusterRecycler/app/src/main/java/com/e/blockbusterrecycler/MovieListViewHolder.kt

```diff
@@ -0,0 +1,18 @@
+package com.e.blockbusterrecycler
+
+import android.view.View
+import android.widget.ImageView
+import android.widget.TextView
+import androidx.recyclerview.widget.RecyclerView
+
+
+class MovieListViewHolder(view: View): RecyclerView.ViewHolder(view) {
+
+    private val movieImage = itemView.findViewById<ImageView>(R.id.imageMovie)
```

Comment:
Be sure to watch out for findViewById again, this is a giveaway to deprecated code. 
`
private val movieImage = imageMovie
`
instead. Then you can press alt+enter and it will import the activity.